rewrite-weblogic/README.md

```diff
@@ -104,11 +104,11 @@ mvn -U org.openrewrite.maven:rewrite-maven-plugin:run \
 ```
 
 > [!TIP]  
-> When upgrading your application to Spring Framework 6.2.x, you may have other libraries that need to be updated, like `org.hibernate`. You can include the recipe in the same command or run the recipe separately, later. For other examples, see the [tutorials](https://github.com/oracle-samples/weblogic-examples).
+> When upgrading your application to Spring Framework 6.2.x, you may have other libraries that need to be updated. You can include the recipe in the same command, like `org.hibernate`, or run other third-party libraries separately. For other examples, see the [tutorials](https://github.com/oracle-samples/weblogic-examples).
```

Comment:
I think worth to mention that we included/created a recipe for hibernate because is commonly used. https://github.com/oracle/rewrite-recipes/blob/c0f8c0166fa69ef96112b3009303de91c23a0975/rewrite-weblogic/src/main/resources/META-INF/rewrite/jakarta-ee-9.1.yaml#L321-348

Reply:
Will do.